css/main.css

```diff
@@ -371,3 +375,39 @@ div.loading {
 	0% { transform: rotate(0deg); }
 	100% { transform: rotate(360deg);}
 }
+
+
+.btn-famnm {
+	background-color: #ffd536;
+	color: black;
+}
+
+.btn-famnm:hover {
+	background-color: #ffe582;
+}
+
+.btn-famnm-secondary {
+	background-color: #3b8ac3;
+	color: white;
+}
+
+.btn-famnm-secondary:hover {
```

Comment:
{nit} put ```famnm``` first since ```btn-``` is bootstrap standard. Putting famnm first would make it more obvious that this is a custom class that we made.

Reply:
good idea; i'll change this to `famnm-btn` and `famnm-btn-secondary`